generated/go/porter/v1/addons.pb.go

```diff
@@ -2078,7 +2078,7 @@ func file_porter_v1_addons_proto_rawDescGZIP() []byte {
 
 var file_porter_v1_addons_proto_enumTypes = make([]protoimpl.EnumInfo, 2)
 var file_porter_v1_addons_proto_msgTypes = make([]protoimpl.MessageInfo, 17)
-var file_porter_v1_addons_proto_goTypes = []interface{}{
+var file_porter_v1_addons_proto_goTypes = []any{
```

Comment:
I installed the latest version of `protoc-gen-go` and it introduced these changes. They only change `interface{}` to `any`, which is cleaner. I think we should keep the tooling up to date, but if this is not desired let me know and I can remove them.